lib/railway_ipc/application.ex

```diff
@@ -40,6 +40,13 @@ defmodule RailwayIpc.Application do
     ]
   end
 
+  def children(true, true, :e2e) do
+    [
+      @repo,
+      {RailwayIpc.Connection.Supervisor, []}
+    ]
```

Comment:
@Lukeghenco this is what we were missing yesterday, the connection process was alive, but it was not a part of the application's supervisor tree.

Reply:
haha, I knew it. Weird this worked in your other test repo though.

Reply:
Yeah, I'm not sure why. The other test repo doesn't use supervisors at all.